.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@4.6.0
+
+executor: &executor
+  executor:
+    name: node/default
+
+commands:
+  install_packages:
+    description: "Installing packages..."
+    steps:
+      - node/install-packages:
+          pkg-manager: yarn 
+  build:
+    description: "Building project..."
+    steps:
+      - run: yarn build
+  test:
+    description: "Testing with BrowserStack..."
+    steps:
+      - run: yarn test-full
+
+workflows:
```

Comment:
Maybe send workflows to bottom of files similar to other mobile sdk? :p
IMO, it's more naturally read: commands -> jobs -> workflows